Format headers of threat classes in  wcf security considerations for data doc (#21217)
dotnet/docs: docs/framework/wcf/feature-details/security-considerations-for-data.md

> When dealing with data in Windows Communication Foundation (WCF), you must consider a number of threat categories. The following table lists the most important threat classes that relate to data processing. WCF provides tools to mitigate these threats.

Reviewer: ```suggestion
When dealing with data in Windows Communication Foundation (WCF), you must consider a number of threat categories. The following list shows the most important threat classes that relate to data processing. WCF provides tools to mitigate these threats.
```

> **Denial of service**

Reviewer: ```suggestion
* Denial of service
```

> When receiving untrusted data, the data may cause the receiving side to access a disproportionate amount of various resources, such as memory, threads, available connections, or processor cycles by causing lengthy computations. A denial-of-service attack against a server may cause it to crash and be unable to process messages from other, legitimate clients.

Reviewer: ```suggestion
  When receiving untrusted data, the data may cause the receiving side to access a disproportionate amount of various resources, such as memory, threads, available connections, or processor cycles by causing lengthy computations. A denial-of-service attack against a server may cause it to crash and be unable to process messages from other, legitimate clients.
```

> **Malicious code execution**

Reviewer: ```suggestion
* Malicious code execution
```

> Incoming untrusted data causes the receiving side to run code it did not intend to.

Reviewer: ```suggestion
  Incoming untrusted data causes the receiving side to run code it did not intend to.
```

> **Information disclosure**

Reviewer: ```suggestion
* Information disclosure
```

> The remote attacker forces the receiving party to respond to its requests in such a way as to disclose more information than it intends to.

Reviewer: ```suggestion
  The remote attacker forces the receiving party to respond to its requests in such a way as to disclose more information than it intends to.
```
